pkg/aws/core.go

```diff
@@ -42,7 +42,10 @@ const (
 	resourceTypeInstance = "instance"
 	resourceTypeVolume   = "volume"
 	// awsEBSDriverName is the name of the CSI driver for EBS
-	awsEBSDriverName = "ebs.csi.aws.com"
+	awsEBSDriverName                    = "ebs.csi.aws.com"
+	awsPlacementAffinity         string = "machine.sapcloud.io/awsPlacementAffinity"
+	awsPlacementAvailabilityZone string = "machine.sapcloud.io/awsPlacementAvailabilityZone"
+	awsPlacementTenancy          string = "machine.sapcloud.io/awsPlacementTenancy"
```

Comment:
```suggestion
	awsPlacementAffinity                = "machine.sapcloud.io/awsPlacementAffinity"
	awsPlacementAvailabilityZone = "machine.sapcloud.io/awsPlacementAvailabilityZone"
	awsPlacementTenancy              = "machine.sapcloud.io/awsPlacementTenancy"
```
String is implicit, we can ignore it.

Reply:
@prashanth26  I had used the `commit suggestion` button here to commit this change you suggested and new commit was made(commit 1)..after that I pushed my commit with other changes from local repo but there I hadn't made these suggested changes(commit2)
I thought the two commits will be different and both will be present here, but commit1 was removed and commit2 remained... I would be happy to know how did it happen
Anyways I have made the changes and will make a new commit with all of them